### PR TITLE
Print logs to debug shutdown integration test

### PR DESCRIPTION
**Pull request by 2opremio**
to track down #1889.

### CODE REVIEW

**Comment by 2opremio** [comment]
And of course, now the test passes. Sigh ... 

Will rebuilt one more time and just merge if it passes again.

**Comment by 2opremio** [comment]
It failed on the second attempt:  https://circleci.com/gh/weaveworks/scope/5357?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
>>> Test ./110_shutdown_test.sh finished after 60.1 secs with error: exit status 1
Check scope exits cleanly within 10 seconds
2546fa33c2dd9c04da13e0a49457ab445141411785ec8f0ca5c940ca31da4115
Weave Scope is reachable at the following URL(s):
  * http://10.240.0.6:4040/
<probe> INFO: 2016/09/23 16:18:23.923461 command line args: --logtostderr=true --mode=probe --probe.docker=true
<probe> INFO: 2016/09/23 16:18:23.923672 probe starting, version 7fddaa9, ID 340ff198d57020ff
<probe> INFO: 2016/09/23 16:18:23.923716 publishing to: localhost:4040
<probe> ERRO: 2016/09/23 16:18:23.951093 Error fetching app details: Get http://127.0.0.1:4040/api: dial tcp 127.0.0.1:4040: getsockopt: connection refused
<probe> WARN: 2016/09/23 16:18:23.985040 Cannot resolve scope.weave.local.: dial tcp 172.17.0.1:53: getsockopt: connection refused
<probe> WARN: 2016/09/23 16:18:23.985305 Error collecting weave status, backing off 10s: Get http://127.0.0.1:6784/report: dial tcp 127.0.0.1:6784: getsockopt: connection refused
<probe> INFO: 2016/09/23 16:18:24.137258 Success collecting weave ps
INFO: 2016/09/23 16:18:24.339891 === received SIGINT/SIGTERM ===
*** exiting
<probe> INFO: 2016/09/23 16:18:24.340508 probe exiting
test #1 "docker_on host1-5357-0.us-central1-a.scope-integration-tests logs weavescope 2>&1 | grep 'app exiting' || (docker_on host1-5357-0.us-central1-a.scope-integration-tests logs weavescope 2>&3 ; false)" failed:
	program terminated with code 1 instead of 0
1 of 3 tests failed in 59.903s.
```

It seems like the app didn't start at all?

**Comment by rade** [comment]
isn't this just a race between
```
scope_on $HOST1 launch
scope_on $HOST1 stop
```
in the test?

Launching is async since runs `docker run -d ...`, so `stop` can come along before `runsvinit` has started one or both of the processes.

A short sleep would 'cure' this.

FWIW, in weave net, `launch` waits until a status http requests on the container succeeds, or the container stops.

**Comment by 2opremio** [comment]
> isn't this just a race between
> ```
> scope_on $HOST1 launch
> scope_on $HOST1 stop
> ```

Good point, I will add the sleep and try to reproduce.

**Comment by 2opremio** [comment]
And as usual @rade was right and the sleep fixed the problem (three runs without failing).

I want to leave the debug output for the future.

@rade PTAL

File: integration/110_shutdown_test.sh [hunk 1]
```diff
@@ -5,12 +5,15 @@
 start_suite "Check scope exits cleanly within 10 seconds"
 
 scope_on $HOST1 launch
+sleep 10
```
Comment (rade) [inline integration/110_shutdown_test.sh]: Does this piece of string have to be as long as 10? I'd try it with 3.

(likewise for the one below)

Comment (rade) [inline integration/110_shutdown_test.sh]: I thought 3 would be the conservative option; the non-conservative one being 1 :)

Anyway, LGTM

**Pull request merged**